.husky/post-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run build && git add dist
```

Comment:
Also wait sorry should this have a `git commit` command also?

Reply:
Nope, because it's in the post-commit hook it gets added to the current commit.

Reply:
Ohhhh nice!